mod_reforged/hooks/entity/tactical/enemies/bandit_leader.nut

```diff
@@ -101,6 +113,73 @@
 		}
 	}
 
+	q.makeMiniboss = @() function()
+	{
+		if (!this.actor.makeMiniboss())
+		{
+			return false;
+		}
+
+		this.getSprite("miniboss").setBrush("bust_miniboss");
+		local shields = clone ::Const.Items.NamedShields;
+		shields.extend([
+			"shields/named/named_bandit_kite_shield",
+			"shields/named/named_bandit_heater_shield"
+		]);
+
+		local r = ::Math.rand(1, 100);
+		if (r <= 25)
+		{
+			local weapon = ::MSU.Class.WeightedContainer([
+				[1, "scripts/items/weapons/named/named_axe"],
+				[1, "scripts/items/weapons/named/named_cleaver"],
+				[1, "scripts/items/weapons/named/named_spear"],
+				[1, "scripts/items/weapons/named/named_sword"],
+				[1, "scripts/items/weapons/named/named_hammer"],
+				[1, "scripts/items/weapons/named/named_mace"],
+
+				[1, "scripts/items/weapons/named/named_rf_kriegsmesser"],
+				[1, "scripts/items/weapons/named/named_rf_swordstaff"],
+				[1, "scripts/items/weapons/named/named_two_handed_flail"],
+				[1, "scripts/items/weapons/named/named_two_handed_mace"],
+				[1, "scripts/items/weapons/named/named_greatsword"]
+			]).roll();
+
+			this.m.Items.equip(::new(weapon));
+		}
+		else if (r <= 50)
```

Comment:
The change from 40 to 50 is a balance change and shouldn't be part of the fix commit.